modules/install-sno-ibm-power.adoc

```diff
@@ -0,0 +1,194 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+:_content-type: PROCEDURE
+[id="installing-sno-on-ibm-power_{context}"]
+= Installing {sno} with {ibmpowerProductName}
+
+.Prerequisites
+
+* You need to setup bastion.
++
+--
+** PXE is used for SNO installation. It requires the following services to be configured and run:
+*** DNS to define api, api-int and *.apps
+*** DHCP service to enable PXE and assign IP to SNO node
+*** HTTP to provide ignition and RHCOS rootfs image
+*** TFTP to enable PXE
+** You will need to install dnsmasq to support DNS, DHCP and PXE, httpd for HTTP.
+--
++
+Perform the following steps to configure bastion to meet these requirements:
++
+. Use the following command to install `grub2`, which is required to enable PXE for PowerVM:
++
+[source,terminal]
+----
+grub2-mknetdir --net-directory=/var/lib/tftpboot
+----
++
+.Example `/var/lib/tftpboot/boot/grub2/grub.cfg` file
+[source,terminal]
+----
+default=0
+fallback=1
+timeout=1
+if [ ${net_default_mac} == fa:b0:45:27:43:20 ]; then
+menuentry "CoreOS (BIOS)" {
+   echo "Loading kernel"
+   linux "/rhcos/kernel" ip=dhcp rd.neednet=1 ignition.platform.id=metal ignition.firstboot coreos.live.rootfs_url=http://192.168.10.5:8000/install/rootfs.img ignition.config.url=http://192.168.10.5:8000/ignition/sno.ign
+   echo "Loading initrd"
+   initrd  "/rhcos/initramfs.img"
+}
+fi
+----
+. Use the following commands to download RHCOS image files from the mirror repo for PXE:
```

Comment:
This is too many commands for one step, and I think that it needs split into multiple steps. You could combine some, but there are 6 commands in this one "Step". I would suggest something like the following: 

. Export the image by. . .
$ export RHCOS_URL=https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/4.12/latest/

. Navigate to the /var/lib/tftpboot/rhcos directy by entering the following command: 
$ cd /var/lib/tftpboot/rhcos

Reply:
Resolved.